File: AtCoder/abc200/B.kt
fun main() {
    val nk = readLine()?.split(" ")?.map { it.toLong() } ?: emptyList()
    var (n, k) = arrayOf(nk[0], nk[1])
    while (k > 0) {
        if (n % 200 == 0L) {
            n /= 200
        } else {
            val s = n.toString() + "200"
            n = s.toLong()
        }
        k--
    }
    print(n)
}

File: AtCoder/abc200/B.swift
let s = readLine() ?? ""
let nk = s.split(separator: " ").map({ Int64($0) ?? Int64(0) })
var (n, k) = (nk[0], nk[1])

while (k > 0) {
	if (n % 200 == 0) {
		n /= 200
	} else {
		let s = String(n) + "200"
		n = Int64(s) ?? Int64(0)
	}
	k -= 1
}

print(n)
